src/app/[team]/scrim/[scrimId]/page.tsx

```diff
@@ -180,6 +180,7 @@ export default async function ScrimDashboardPage({ params }: Props) {
                     <CardHeader className="">
                       <h3 className="z-10 text-3xl font-semibold tracking-tight text-white">
                         {map.name}
+                        <div className="text-lg">{map.replayCode ?? ""}</div>
```

Comment:
I would like the replay code to be in the bottom right. My vision for the replay code looks like this:
![CleanShot 2024-08-22 at 23 34 08@2x](https://github.com/user-attachments/assets/92f06cc5-c6dc-428f-aa84-103d378ee933)
To achieve this, add a `CardFooter` with the following code:
```tsx
<CardFooter className="float-right flex items-center justify-between pt-10">
  <div className="z-10 font-semibold tracking-tight text-white">
    {map.replayCode ?? ""}
  </div>
</CardFooter>
```